package.json

```diff
@@ -93,26 +93,24 @@
   "entry": "lib/index.js",
   "types": "lib/index.d.ts",
   "devDependencies": {
-    "@chanzuckerberg/axe-storybook-testing": "^5.0.1",
+    "@chanzuckerberg/axe-storybook-testing": "^6.0.0",
```

Comment:
FYI, I'm going to release v6.0.1 today. Don't need to wait for it or anything, unless you want to make sure this is on the latest and greatest.